functions.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/** Дедлайн */
+
+function get_time($final_data):int {
+    $HOURS = 3600;
+    $now_date = time();
+    $diff_time = strtotime($final_data) - $now_date;
+    $hours_until_end = floor($diff_time / $HOURS);
+    return $hours_until_end;
+};
+
+/** Подсчет проектов */
+
+function do_counting(string $name, array $items):int {
+   $number = 0;
+   foreach($items as $task) {
+       if($name == $task['project']) {
+           $number ++;
+       }
+   };
+   return $number;
+};
+
+/** Фильтрация */
+
+function esc($str) {
+   $text = htmlspecialchars($str);
+   return $text;
+};
+
+/** Проверяем наличие ошибки соединения */
+
+function find_mistake(string $error):string {
+    $content = include_template('error.php',['error' => $error]);
+    return $content;
+};
+
+/** Текст ошибки */
+
+function data_mistake_text(string $error) {
+    $content = find_mistake($error);
+    $layout_content = include_template('layout.php', [
+        'content' => $content,
+        'title' => 'Дела в порядке'
+      ]);
+
+    print($layout_content);
+    exit;
+};
+
+
+/** Подключение к БД */
+
+function do_connection():object {
+    $connection = mysqli_connect('localhost', 'root', '', 'doings');
+    mysqli_set_charset($connection, 'utf8');
+    if (!$connection) {
+        $error = mysqli_connect_error();
+        data_mistake_text($error);
+    }
+    return $connection;
+};
+
+/** Получаем список пользователей */
+
+function find_users(object $connection):array {
+ /** Приводим id к числовому типу */
```

Comment:
Инлайн комментарии делаем просто с `//`
```suggestion
    // Приводим id к числовому типу
```